content/news/2022_02_24_GRASS_GIS_8_0_1_released.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "GRASS GIS 8.0.1 released"
+date: 2022-02-24T21:38:40+02:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### What's new in a nutshell
+
+<!--- screenshot plus small description - TBD --->
+
+#### Full list of changes and new features
+
+The overview of new features in the 8.0 release series is available at
+[new features in GRASS GIS 8.0](https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80).
+See also our detailed announcement with the full list of changes and
+bugs fixed at
+[https://github.com/OSGeo/grass/releases/tag/8.0.1](https://github.com/OSGeo/grass/releases/tag/8.0.1).
+
+## Download and test!
+
+#### Binaries/Installers download
+
+- winGRASS 8.0.1/standalone:
+  [64bit standalone installer](/grass80/binary/mswindows/native/WinGRASS-8.0.1-1-Setup.exe)
+- winGRASS 8.0.1/OSGeo4W:
+  [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
+- Mac
+    - [Official Application Bundle](http://grassmac.wikidot.com/downloads)
+    - [MacPorts](https://ports.macports.org/port/grass/)
```

Comment:
```suggestion
    - [MacPorts](https://ports.macports.org/port/grass/)
```

This link says:
```
grass v 6.4.6
Obsolete port, replaced by grass7
```

Perhaps to be removed? Or does e.g. @nilason have an idea how to get a fresh macports version?

Reply:
You should remove it. No G8 on macports yet, PR stalled (for me) unknown reason (https://github.com/macports/macports-ports/pull/13801).